### DISCUSSION TITLE
Authentication with LDAP/AD

### DISCUSSION THREAD

**Discussion by ryzam** [Budibase/budibase#3894] [Ideas]
Hi guys, its normal for any internal apps to authenticate using their current LDAP/AD server. It would nice if you can plan to support this in the future release.

**Comment by shogunpurple** [comment]
Hi @ryzam - is this not something that can be set up through our SSO/OpenID connect integration?

https://docs.budibase.com/building-apps/admin-and-management/authentication

**Reply by yuppox** [reply]
That link is broken.

I would also like support for LDAP. I'm not sure about the SSO/OpenID. I just need it to connect to LDAP on localhost.

**Comment by oldwired** [comment]
For internally used self hosted tools it would be really useful if we could use the local LDAP/Active Directory for authentication (like with self hosted gitlab for example).
But I understand it may not fit into your product because I think there won't be much use for this in the cloud hosted version.

**Comment by MacDork** [comment]
Any chance of reconsidering this request?
